### PR TITLE
Bump version to v0.2.25

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

**Comment by zanieb** [comment]
Don't want to wait until your docs publish pipeline is done? :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## 0.2.25
+
+### Enhancements
+
+- Add `pypy` executables when calling `uv venv` ([#5047](https://github.com/astral-sh/uv/pull/5047))
```
Comment (zanieb) [inline CHANGELOG.md:7]:
```suggestion
- Include PyPy-specific executables when creating virtual environments with `uv venv` ([#5047](https://github.com/astral-sh/uv/pull/5047))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+- Fall back to streaming wheel when `Content-Length` header is absent ([#5000](https://github.com/astral-sh/uv/pull/5000))
```
Comment (zanieb) [inline CHANGELOG.md:8]: Arguably a bug fix

**Review by zanieb — APPROVED** [review]

File: PREVIEW-CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+- Add documentation for running scripts ([#4968](https://github.com/astral-sh/uv/pull/4968))
+- Add guide for tools ([#4982](https://github.com/astral-sh/uv/pull/4982))
+- Allow URL dependencies in tool run `--from` ([#5002](https://github.com/astral-sh/uv/pull/5002))
+- Add guide for authenticating to Azure Artifacts ([#4857](https://github.com/astral-sh/uv/pull/4857))
+- Improve rc file detection based on rustup ([#5026](https://github.com/astral-sh/uv/pull/5026))
+- Rename `python install --force` parameter to `--reinstall` ([#4999](https://github.com/astral-sh/uv/pull/4999))
+- Use lockfile to prefill resolver index ([#4495](https://github.com/astral-sh/uv/pull/4495))
+- `uv tool install` hint the correct when the executable is available ([#5019](https://github.com/astral-sh/uv/pull/5019))
+- `uv tool run` error messages references `uvx` when appropriate ([#5014](https://github.com/astral-sh/uv/pull/5014))
+- `uvx` warn when executable is not part of `--from PKG` ([#5071](https://github.com/astral-sh/uv/pull/5071))
```
Comment (zanieb) [inline PREVIEW-CHANGELOG.md:16]:
```suggestion
- `uvx` warns when requested executable is not provided by the package [#5071](https://github.com/astral-sh/uv/pull/5071))
```

**Comment by charliermarsh** [comment]
Haha. I actually intentionally wanted to release before :D

**Pull request merged**